frontend/src/components/podcast/episode-table/episode-table.tsx

```diff
@@ -0,0 +1,102 @@
+import { useMemo, useTable } from 'hooks/hooks';
+import { Column } from 'react-table';
+
+import { EpisodeItem } from '../common/types/types';
+import styles from './styles.module.scss';
+
+type Props = {
+  episodes?: EpisodeItem[];
+};
+
+type EpisodeDescription = {
+  episodeName: string;
+  authorName: string;
+};
+
+const EpisodeTable: React.FC<Props> = ({ episodes = [] }) => {
+  const data: EpisodeItem[] = useMemo(() => episodes, [episodes]);
+
+  const columns: Column<EpisodeItem>[] = useMemo(
+    () => [
+      {
+        Header: '#',
+        accessor: (_originalRow, rowIndex): string => String(rowIndex),
+      },
+      {
+        Header: 'Name and Host',
+        accessor: (row: EpisodeItem): EpisodeDescription => {
+          return {
+            episodeName: row.episodeName,
+            authorName: `${row.userFirstName} ${row.userLastName}`,
+          };
+        },
+        Cell: function myFunc({
```

Comment:
It will be better if you call 'myFunc' function something like 'Cellitem'. The name of this function will be used in Developer Tools, and will indicate specific Cell component.